test/Groups/elements.jl

```diff
@@ -1,3 +1,54 @@
+Oscar.@_AuxDocTest "show and print fp group elements", (fix = false),
+raw"""
+common setup
+
+```jldoctest FPGroupElem.show
+julia> using Oscar
+
+julia> F = free_group('a':'z');
+
+julia> w1 = prod(gens(F));
+
+julia> w2 = prod(gens(F))*prod(inv.(gens(F)));
+```
+
+default `show` without unicode
+
+```jldoctest FPGroupElem.show
+julia> old = allow_unicode(true; temporary=false);
```

Comment:
`true` means *with* unicode

Reply:
```suggestion
julia> old = allow_unicode(false; temporary=true);
```
I think you want this (no unicode, but only set temporary as you also only revert it temporary afterwards

Reply:
Yeah that's a copy&paste mistake, thank you for catching that.

(Hrm... now I wonder how persistent those local Julia prefs are on our runners and if we maybe need to explicitly reset them at the start of CI runs or else get weirdness... CC @aaruni96)

Reply:
this shouldn't be needed as https://github.com/oscar-system/Oscar.jl/blob/5cac5089a48adf23dd111da84ed3b8ec69cce63a/src/utils/tests.jl#L207 makes sure to run tests without unicode enabled